container/multi-value-slice/interfaces/interfaces.go

```diff
@@ -0,0 +1,8 @@
+package interfaces
```

Comment:
I had to create a new package to avoid circular dependencies. Types defined here are used both in tests and in `assert`/`require` packages